charts/app-config/templates/dex.yaml

```diff
@@ -0,0 +1,23 @@
+{{ if has .Values.govukEnvironment (list "integration") }}
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: dex
+  namespace: {{ .Values.argoNamespace | default .Release.Namespace }}
+spec:
+  project: default
+  source:
+    repoURL: https://charts.dexidp.io
+    chart: dex
+    targetRevision: "0.9.0"
+    helm:
+      values: |
+        {{- include "app-config.dex-values" . | nindent 8 }}
+  destination:
+    server: https://kubernetes.default.svc
+    namespace: {{ .Values.argoNamespace }}
+  syncPolicy:
+    automated: null
```

Comment:
Just checking this is intentional? (no probs if so, just mentioning in case you'd left it in unintentionally)

Reply:
Yep, setting this to null disables auto-sync so it won't make any changes if I've made a mistake in copying the config over